src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.reader.CsvFileReader;
+import core.basesyntax.report.CsvFileWriter;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvFileReader csvFileReader = new CsvFileReader("testData1.csv");
+        Map<String, Integer> fruitQuantities = csvFileReader.readTransactions();
```

Comment:
Make your services independent and call them in the `main()` method. The `CsvFileReader` should not directly return a `Map` of fruit quantities; instead, it should return the raw data which another service can process.

Reply:
Make your services independent and call them in the main() method. The CsvFileReader should not directly return a Map of fruit quantities; instead, it should return the raw data which another service can process. - agreed. Those should be separate steps. First, get raw data. Then use another service to work with it